site2/website/release-notes.md

```diff
@@ -797,6 +797,116 @@ For more context about this issue, see [[Pulsar Functions and IO] Cannot upgrade
+- Call .release() when discarding entry to prevent direct memory leak. [#11748](https://github.com/apache/pulsar/pull/11748)
+- Fix the topic in fenced state and can not recover. [#11737](https://github.com/apache/pulsar/pull/11737)
+- Support disabling non-TLS service ports. [#11724](https://github.com/apache/pulsar/pull/11724)
+- fix the bug, can not update topic when the update topicName is contained by an existed topic as a part. [#11686](https://github.com/apache/pulsar/pull/11686)
+- Fix Pulsar didn't respond error messages when throw InterceptException. [#11650](https://github.com/apache/pulsar/pull/11650)
+- Avoid redundant calls for getting the offload policies from the offloader. [#11629](https://github.com/apache/pulsar/pull/11629)
+- Optimize ManagedLedger Ledger Ownership Check. [#13222](https://github.com/apache/pulsar/pull/13222)
+- Do not reuse the Failed OpAddEntry object which lead bundle unloading timeout. [#12993](https://github.com/apache/pulsar/pull/12993)
+- Fix producer getting incorrectly removed from topic's producers map. [#12846](https://github.com/apache/pulsar/pull/12846)
+- Allow to configure schema compatibility policy for system topics. [#12598](https://github.com/apache/pulsar/pull/12598)
+- Cleanup already deleted namespace topics. [#12583](https://github.com/apache/pulsar/pull/12583)
+- Add OpAddEntry to pendingAddEntries after the state check. [#12570](https://github.com/apache/pulsar/pull/12570)
+- Cancel scheduled tasks when deleting ManagedLedgerImpl. [#12565](https://github.com/apache/pulsar/pull/12565)
+- NPE on OpAddEntry while ManagedLedger is closing. [#12364](https://github.com/apache/pulsar/pull/12364)
+- Avoid potentially blocking calls to metadata on critical threads. [#12340](https://github.com/apache/pulsar/pull/12340)
+- Fixed used after recycle issue in OpAddEntry. [#12103](https://github.com/apache/pulsar/pull/12103)
+- Optimize the memory usage of Cache Eviction. [#12045](https://github.com/apache/pulsar/pull/12045)
+- Fix wrong key-hash selector used for new consumers after all the previous consumers disconnected. [#12035](https://github.com/apache/pulsar/pull/12035)
+- Only auth_errors should log at error level. [#9325](https://github.com/apache/pulsar/pull/9325)
+
+#### Dependency upgrade
+- Upgrade Log4j2 version to 2.14.0 and replace legacy log4j dependency. [#8880](https://github.com/apache/pulsar/pull/8880)
+- Remove Boost::System runtime dependency. [#9498](https://github.com/apache/pulsar/pull/9498)
+- Upgrade commons-compress to 1.21. [#11345](https://github.com/apache/pulsar/pull/11345)
+- Upgrade Jetty to 9.4.43.v20210629. [#11660](https://github.com/apache/pulsar/pull/11660)
+- Add maven.restlet.org repository. [#13248](https://github.com/apache/pulsar/pull/13248)
+- Bump log4j to 2.15.0. [#13226](https://github.com/apache/pulsar/pull/13226)
```

Comment:
for 2.7.4, it should be 2.16.0?

Reply:
And it's better to create use a new section to highlight the security issue fixes.

Reply:
Ok.